splink/comparison_creator.py

```diff
@@ -56,6 +90,55 @@ def create_comparison_dict(self, sql_dialect_str: str) -> dict:
     def input_column(self, sql_dialect: SplinkDialect) -> InputColumn:
         return InputColumn(self.col_name, sql_dialect=sql_dialect.sqlglot_name)
 
+    @final
+    def configure(
+        self,
+        *,
+        m_probabilities: list[float] = None,
+        u_probabilities: list[float] = None,
+    ) -> "ComparisonCreator":
+        self.m_probabilities = m_probabilities
+        self.u_probabilities = u_probabilities
+        return self
```

Comment:
Was experimenting with docstrings to see how they appeared in intellisense.  Came up with this as a starter for 10.  It works with intellisense nicely

```
        """Configure the comparison creator with m and u probabilities. The first
        element in the list corresponds to the first comparison level, usually
        an exact match level. Subsequent elements correspond comparison 
        levels in sequential order, through to the last element which usually 
        the 'ELSE' level.

        Example:
            cc = LevenshteinAtThresholds("name", 2)
            cc.configure(
                m_probabilities=[0.9, 0.08, 0.02],
                u_probabilities=[0.01, 0.05, 0.94]
            )
        """
```